CONTRIBUTING.md

```diff
@@ -17,6 +17,74 @@ The pull requests will be reviewed and eventually merged into the main repo. See
 
 In order to setup a local environment, please follow the instructions in the [README](./README.md).
 
+## Rust and Cairo-Foundry good practices
+
+-   Making use of [clippy](https://github.com/rust-lang/rust-clippy):
+
+    > A collection of lints to catch common mistakes and improve your Rust code.
+
+    -   Update your Rust version: run `rustup update`
+    -   Install clippy: run `rustup component add clippy`
+    -   Run clippy and receive some tips: run `cargo clippy`
+
+-   `expect()` and `unwrap()`:
+
+    -   You should avoid using these methods as much as possible as they make the current thread panic. As a general rule, we want to handle error cases.
+    -   There are some rare cases where `expect()` and `unwrap()` suitable. Then, comment your code and share your reasoning with your co-contributors: why did you need and properly use these methods.
```

Comment:
where expect() and unwrap() `are` suitable

Reply:
okkay